roles/nova-common/templates/etc/serverspec/nova_spec.rb

```diff
@@ -55,10 +55,10 @@
 
 describe file('/etc/nova/nova.conf') do
   it { should contain 'debug = {{ nova.logging.debug }}' }
-end 
+end
 
-describe file( '{{ nova.state_path }}/instances' ) do
+has_file = file('{{ nova.state_path }}/instances').exists?
+describe file( '{{ nova.state_path }}/instances' ), :if => has_file do
   it { should exist }
   it { should be a directory }
```

Comment:
This test makes little sense to me now. You're seeing if the file exists, in order to see if the file exists? I guess you're checking to make sure it's a directory instead of a file, but is that really a thing we need to check for?